Reason: "the traceback is not preserved, exarkun said he'll try to fix this! god knows how"
Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_failure.py", line 94, in testLackOfTB
    self.assertEquals(innerline, '1/0')
twisted.trial.unittest.FailTest: not equal:
a = 'raise self.type, self.value, self.tb'
b = '1/0'

===============================================================================
[TODO]: twisted.test.test_modules.PathModificationTest.test_underUnderPathNotAlreadyImported

Reason: 'This may be impossible but it sure would be nice.'
Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_modules.py", line 270, in test_underUnderPathNotAlreadyImported
    self._underUnderPathTest(False)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_modules.py", line 254, in _underUnderPathTest
    importPackages=doImport),
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_modules.py", line 37, in findByIteration
    self.fail("Unable to find module %r through iteration." % (modname,))
twisted.trial.unittest.FailTest: Unable to find module 'pyspacetests3.foozle' through iteration.
===============================================================================
[TODO]: twisted.test.test_modules.RebindingTest.test_underUnderPathNotAlreadyImported

Reason: 'This may be impossible but it sure would be nice.'
Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_modules.py", line 270, in test_underUnderPathNotAlreadyImported
    self._underUnderPathTest(False)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_modules.py", line 254, in _underUnderPathTest
    importPackages=doImport),
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_modules.py", line 37, in findByIteration
    self.fail("Unable to find module %r through iteration." % (modname,))
twisted.trial.unittest.FailTest: Unable to find module 'pyspacetests7.foozle' through iteration.
===============================================================================
[TODO]: twisted.test.test_modules.ZipPathModificationTest.test_underUnderPathNotAlreadyImported

Reason: 'This may be impossible but it sure would be nice.'
Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_modules.py", line 270, in test_underUnderPathNotAlreadyImported
    self._underUnderPathTest(False)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_modules.py", line 254, in _underUnderPathTest
    importPackages=doImport),
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_modules.py", line 37, in findByIteration
    self.fail("Unable to find module %r through iteration." % (modname,))
twisted.trial.unittest.FailTest: Unable to find module 'pyspacetests11.foozle' through iteration.
===============================================================================
[FAIL]: twisted.test.test_tcp.HalfCloseBuggyApplicationTests.test_readNotificationRaises

Traceback (most recent call last):
  File "test_tcp.py", line 1739, in check
    
twisted.trial.unittest.FailTest: not equal:
a = 0
b = 1

===============================================================================
[ERROR]: twisted.test.test_iutils.ProcessUtilsTests.test_outputWithErrorIgnored

Traceback (most recent call last):
Failure: twisted.internet.defer.TimeoutError: <twisted.test.test_iutils.ProcessUtilsTests testMethod=test_outputWithErrorIgnored> (test_outputWithErrorIgnored) still running at 120.0 secs
===============================================================================
[ERROR]: twisted.test.test_iutils.ProcessUtilsTests.test_outputWithErrorIgnored

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 201, in _doWriteOrRead
    why = selectable.doRead()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 240, in doRead
    return fdesc.readFromFD(self.fd, self.dataReceived)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/fdesc.py", line 94, in readFromFD
    callback(output)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 243, in dataReceived
    self.proc.childDataReceived(self.name, data)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 760, in childDataReceived
    self.proto.childDataReceived(name, data)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/protocol.py", line 548, in childDataReceived
    self.errReceived(data)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/utils.py", line 79, in errReceivedIsBad
    self.transport.loseConnection()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 718, in loseConnection
    self.closeStdin()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 709, in closeStdin
    self.closeChildFD(0)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 692, in closeChildFD
    self.pipes[childFD].loseConnection()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 252, in loseConnection
    self.startWriting()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 286, in startWriting
    self.reactor.addWriter(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 133, in addWriter
    self._updateRegistration(fd, select.KQ_FILTER_WRITE, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 32] Broken pipe
===============================================================================
[ERROR]: twisted.test.test_iutils.ProcessUtilsTests.test_outputWithErrorIgnored

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/python/log.py", line 84, in callWithLogger
    return callWithContext({"system": lp}, func, *args, **kw)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/python/log.py", line 69, in callWithContext
    return context.call({ILogContext: newCtx}, func, *args, **kw)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/python/context.py", line 59, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/python/context.py", line 37, in callWithContext
    return func(*args,**kw)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 212, in _doWriteOrRead
    self.removeWriter(selectable)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 153, in removeWriter
    self._updateRegistration(fd, select.KQ_FILTER_WRITE, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.testNormalTermination

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1017, in testNormalTermination
    usePTY=self.usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.testNormalTermination

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.testOpeningTTY

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 2024, in testOpeningTTY
    path=None, usePTY=self.usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.testOpeningTTY

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_abnormalTermination

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1037, in test_abnormalTermination
    env=None, usePTY=self.usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_abnormalTermination

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_childSignalHandling

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1125, in test_childSignalHandling
    return self._testSignal(signal.SIGUSR1)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1053, in _testSignal
    usePTY=self.usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_childSignalHandling

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_executionError

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1142, in test_executionError
    usePTY=self.usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_executionError

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_signalHUP

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1065, in test_signalHUP
    return self._testSignal('HUP')
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1053, in _testSignal
    usePTY=self.usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_signalHUP

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_signalINT

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1076, in test_signalINT
    return self._testSignal('INT')
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1053, in _testSignal
    usePTY=self.usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_signalINT

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_signalKILL

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1087, in test_signalKILL
    return self._testSignal('KILL')
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1053, in _testSignal
    usePTY=self.usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_signalKILL

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_signalTERM

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1098, in test_signalTERM
    return self._testSignal('TERM')
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 1053, in _testSignal
    usePTY=self.usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.PosixProcessTestCasePTY.test_signalTERM

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.TestTwoProcessesPosix.testClosePty

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 836, in testClosePty
    self.createProcesses(usePTY=1)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 783, in createProcesses
    usePTY=usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.TestTwoProcessesPosix.testClosePty

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 812, in tearDown
    os.kill(pr.pid, signal.SIGTERM)
exceptions.AttributeError: 'NoneType' object has no attribute 'pid'
===============================================================================
[ERROR]: twisted.test.test_process.TestTwoProcessesPosix.testClosePty

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_process.TestTwoProcessesPosix.testKillPty

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 843, in testKillPty
    self.createProcesses(usePTY=1)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 783, in createProcesses
    usePTY=usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_process.TestTwoProcessesPosix.testKillPty

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_process.py", line 812, in tearDown
    os.kill(pr.pid, signal.SIGTERM)
exceptions.AttributeError: 'NoneType' object has no attribute 'pid'
===============================================================================
[ERROR]: twisted.test.test_process.TestTwoProcessesPosix.testKillPty

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_stdio.StandardInputOutputTestCase.test_consumer

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 122, in eb
    return callback(err)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 311, in processEnded
    self.assertEquals(p.data[1], file(junkPath).read())
exceptions.KeyError: 1
===============================================================================
[ERROR]: twisted.test.test_stdio.StandardInputOutputTestCase.test_hostAndPeer

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 122, in eb
    return callback(err)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 222, in processEnded
    host, peer = p.data[1].splitlines()
exceptions.KeyError: 1
===============================================================================
[ERROR]: twisted.test.test_stdio.StandardInputOutputTestCase.test_lastWriteReceived

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 194, in test_lastWriteReceived
    usePTY=True)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 115, in _spawnProcess
    **kw)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 319, in spawnProcess
    processProtocol, uid, gid, usePTY)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/process.py", line 835, in __init__
    self.startReading()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/abstract.py", line 278, in startReading
    self.reactor.addReader(self)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 124, in addReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_ADD)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 45] Operation not supported
===============================================================================
[ERROR]: twisted.test.test_stdio.StandardInputOutputTestCase.test_lastWriteReceived

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/unittest.py", line 839, in _classCleanUp
    util._Janitor(self, result).postClassCleanup()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 145, in postClassCleanup
    selectables = self._cleanReactor()
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/trial/util.py", line 205, in _cleanReactor
    for sel in reactor.removeAll():
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 161, in removeAll
    [self._selectables[fd] for fd in self._writes])
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/posixbase.py", line 510, in _removeAll
    self.removeReader(reader)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 143, in removeReader
    self._updateRegistration(fd, select.KQ_FILTER_READ, select.KQ_EV_DELETE)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/internet/kqreactor.py", line 115, in _updateRegistration
    self._kq.control([select.kevent(*args)], 0, 0)
exceptions.OSError: [Errno 2] No such file or directory
===============================================================================
[ERROR]: twisted.test.test_stdio.StandardInputOutputTestCase.test_loseConnection

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 122, in eb
    return callback(err)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 139, in processEnded
    for line in file(errorLogFile):
exceptions.IOError: [Errno 2] No such file or directory: 'twisted.test.test_stdio/StandardInputOutputTestCase/test_loseConnection/cUPM2H/temp'
===============================================================================
[ERROR]: twisted.test.test_stdio.StandardInputOutputTestCase.test_producer

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 122, in eb
    return callback(err)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 292, in processEnded
    self.assertEquals(p.data[1], ''.join(written))
exceptions.KeyError: 1
===============================================================================
[ERROR]: twisted.test.test_stdio.StandardInputOutputTestCase.test_producer

Traceback (most recent call last):
Failure: twisted.trial.util.DirtyReactorAggregateError: Reactor was unclean.
DelayedCalls: (set twisted.internet.base.DelayedCall.debug = True to debug)
<DelayedCall 0x1d85760 [0.00967001914978s] called=0 cancelled=0 connectionMade(None)>
===============================================================================
[ERROR]: twisted.test.test_stdio.StandardInputOutputTestCase.test_readConnectionLost

Traceback (most recent call last):
Failure: twisted.internet.error.ProcessTerminated: A process has ended with a probable error condition: process ended with exit code 1.
===============================================================================
[ERROR]: twisted.test.test_stdio.StandardInputOutputTestCase.test_write

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 122, in eb
    return callback(err)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 240, in processEnded
    self.assertEquals(p.data[1], 'ok!')
exceptions.KeyError: 1
===============================================================================
[ERROR]: twisted.test.test_stdio.StandardInputOutputTestCase.test_writeSequence

Traceback (most recent call last):
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 122, in eb
    return callback(err)
  File "/Users/rnadhani/Codes/Twisted/twisted-kqueue/twisted/test/test_stdio.py", line 256, in processEnded
    self.assertEquals(p.data[1], 'ok!')
exceptions.KeyError: 1
===============================================================================
[ERROR]: twisted.test.test_tcp.HalfCloseTestCase.testCloseWriteCloser

Traceback (most recent call last):
Failure: twisted.internet.defer.TimeoutError: <twisted.test.test_tcp.HalfCloseTestCase testMethod=testCloseWriteCloser> (testCloseWriteCloser) still running at 120.0 secs
===============================================================================
[ERROR]: twisted.test.test_tcp.HalfCloseTestCase.testCloseWriteCloser

Traceback (most recent call last):
Failure: twisted.trial.util.DirtyReactorAggregateError: Reactor was unclean.
DelayedCalls: (set twisted.internet.base.DelayedCall.debug = True to debug)
<DelayedCall 0x2937b70 [-8.08238983154e-05s] called=0 cancelled=0 LoopingCall<0>(check, *(), **{})()>
===============================================================================
[ERROR]: twisted.test.test_tcp.HalfCloseTestCase.testCloseWriteCloser

Traceback (most recent call last):
Failure: twisted.trial.util.DirtyReactorAggregateError: Reactor was unclean.
Selectables:
<<class 'twisted.internet.tcp.Port'> of twisted.test.test_tcp.MyHCFactory on 60298>
===============================================================================
[ERROR]: twisted.test.test_tcp.HalfCloseTestCase.testWriteCloseNotification

Traceback (most recent call last):
Failure: twisted.internet.defer.TimeoutError: <twisted.test.test_tcp.HalfCloseTestCase testMethod=testWriteCloseNotification> (testWriteCloseNotification) still running at 120.0 secs
===============================================================================
[ERROR]: twisted.test.test_tcp.HalfCloseTestCase.testWriteCloseNotification

Traceback (most recent call last):
  File "test_tcp.py", line 1583, in tearDown
    
twisted.trial.unittest.FailTest: not equal:
a = 1
b = 0

===============================================================================
[ERROR]: twisted.test.test_tcp.HalfCloseTestCase.testWriteCloseNotification

Traceback (most recent call last):
Failure: twisted.trial.util.DirtyReactorAggregateError: Reactor was unclean.
DelayedCalls: (set twisted.internet.base.DelayedCall.debug = True to debug)
<DelayedCall 0x296a9e0 [-0.000108003616333s] called=0 cancelled=0 LoopingCall<0>(check, *(), **{})()>
-------------------------------------------------------------------------------
Ran 1857 tests in 410.160s

FAILED (skips=59, expectedFailures=4, failures=1, errors=43, successes=1771)
